ivy/stateful/module.py

```diff
@@ -646,8 +699,9 @@ def build(
             True for successfully built a module.
         """
         self._dev = ivy.default(device, self._dev)
+        # build buffers if any
+        self._create_buffers()
```

Comment:
Given that `ivy.Module._create_buffers` has been removed, we should also remove this